packages/validator-ajv8/src/precompiledValidator.ts

```diff
@@ -101,7 +109,7 @@ export default class AJV8PrecompiledValidator<
    * @throws - Error when the schema provided does not match the base schema of the precompiled validator
    */
   rawValidation<Result = any>(schema: S, formData?: T): RawValidationErrorsType<Result> {
-    if (!isEqual(schema, this.rootSchema)) {
+    if (!isEqual(schema, this.resolvedRootSchema)) {
```

Comment:
Does the `isValid()` call get the `rootSchema` or the `resolvedRootSchema`? If you aren't sure, maybe check both?

Reply:
looking at the code it seems like `isValid` is called by the `rootSchema` without being resolved. All the components go through `SchemaUtils` which is initialized with the "raw" schema. So I think its fine to leave isValid using the `rootSchema`